test/testdata/tactic/GoldenIdTypeFam.hs.expected

```diff
@@ -4,4 +4,4 @@ type family TyFam
 type instance TyFam = Int
 
 tyblah' :: TyFam -> Int
-tyblah' = (\ i -> i)
+tyblah' = id
```

Comment:
Looks like the extraneous parens are not caused by the splicer! That means we should be able to eliminate them for all top-level holes!

Reply:
Tackled in #1352